thunder/core/prims.py

```diff
@@ -4102,6 +4103,59 @@ def matmul_meta(a: TensorProxy, b: TensorProxy, /) -> TensorProxy:
     PrimIDs.MATMUL, "matmul", meta=matmul_meta, tags=(OpTags.MATMUL_OP, OpTags.DONT_AUTO_RECOMPUTE_IN_BACKWARD)
 )
 
+
+# TODO: Check `b`'s being column-major.
+def _scaled_mm_meta(
+    a: TensorProxy,
+    b: TensorProxy,
+    scale_a: TensorProxy,
+    scale_b: TensorProxy,
+    bias: TensorProxy | None = None,
+    scale_result: TensorProxy | None = None,
+    out_dtype: dtypes.dtype | None = None,
+    use_fast_accum: bool = False,
+) -> TensorProxy:
+    # NOTE: Currently this only supports fp8 a & b and per-tensor scales.
+    # TODO: Devise a way to make sure `a` is row-major and `b` is column-major.
```

Comment:
You can use `prims.stride_order(a, (1, 0))` for conversion to row-major and `prims.stride_order(b, (0, 1)` for conversion to column-major matrices.

Reply:
I was rather thinking of having a check to see the first two tensors satisfy the condition rather than having `stride_order` because I think callers of `scaled_mm` are responsible for having the inputs conforming to the condition.